license/STATUS.txt

```diff
@@ -0,0 +1,25 @@
+Open Touryo Project Status
+=============================
+
+Project Status
+==============
+Open Touryo is an in-house tool of Hitachi Solutions, Ltd. 
```

Comment:
@daisukenishino 

*is* is the present tense.
Open Touryo is now oss, and is no longer an in-house tool.

So, please revise as follows:  
Open Touryo was created as an in-house tool of Hitachi Solutions, Ltd.